frontend/javascripts/oxalis/view/novel_user_experiences/welcome_toast.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React, { useEffect } from "react";
+import { Button, notification } from "antd";
+import { useSelector } from "react-redux";
+import features from "features";
+
+function showWelcomeToast() {
+  notification.open({
+    message: (
+      <div>
+        <span className="logo" />
+        welcome to webKnossos!
+      </div>
+    ),
+    description: (
+      <div>
+        You are viewing this dataset on webKnossos. webKnossos is an online tool to visualise and
+        collaboratively annotate large image datasets. Try out the annotation features on the public
+        dataset below. If this is for you, sign up for free, upload your data and get started!
```

Comment:
@hotzenklotz any feedback for the final wording?

Reply:
webKnossos is a web-based platform for visualization, annotation, and sharing of large-scale 3D image datasets. Try out the annotation features and upload your own data with a free account.

Reply:
see norman's suggestion below, please